game_api.txt

```diff
@@ -213,6 +213,8 @@ The doors mod allows modders to register custom doors and trapdoors.
 	description = "Door description",
 	inventory_image = "mod_door_inv.png",
 	groups = {choppy = 2},
+	model = "NAME", -- (optional)
+	-- Model name without prefix ("big_door" not "big_door_a.obj", "big_door_b.obj")
```

Comment:
Following the way the other properties are documented, this should rather be something like this:
```
model = "mod_door", -- …
-- …
```

And what you mean here is suffix, not prefix.

Reply:
OK done.